### ISSUE TITLE
Sending plugin values to Nightscout via NFC scan

### ISSUE THREAD

**Issue by berislavv** [NightscoutFoundation/xDrip#1393]
HI
I am using xdrip+ with Blucon on my 6 year son and sending data to Nightscout for remote monitoring.
We are using it for last 2 years and really appreciate your great work on this app.
As his skin is sensitive we are now placing libre on his hip and i have there issues to fix the blucon device.
Also he is complaining lately about it so we will allow him summer without blucon.

So we are currently scaning the Libre only via NFC.
We are also using Datricsae plugin in for calibration and displaying BG level, its more precise for us then standard xdrip which can differ in 40 mg/dl ( 2 mmol/l).

And now when i scan using NFC xdrip send to Nightscout last 15 minutes of data calculated via Datricsae but historical data send caluclated via xdrip standard algorithm.
Send display data to nightscout is enabled.

Is there any way to fix it?
In xdrip i am entering all the treatmens and we use nightscout reports for our diabetologist.
If this cannot be fixed i need to find a workaround for this.

Thanks for any help

Berislav

**Comment by Navid200** [comment]
@berislavv Have you asked in the support channels?
https://www.facebook.com/groups/xDripG5
https://gitter.im/jamorham/xDrip-plus
Your question will be seen by a significantly larger audience.
There may already be solutions, or work-arounds.

**Comment by tolot27** [comment]
Do I understand it correctly that the Datricsae plugin calculates only the last 15 minutes?

**Comment by berislavv** [comment]
No
Datricsae calculates correctly but the value that is sended to Nightscout are from Datricase only last 15 minutes and older are from original Xdrip. In our case datricase was much more precise then xdrip algorithm. 
The result is very jumpy Nightscout graph, xdrip on phone is fine.

**Comment by Navid200** [comment]
@tzachi-dar Any suggestions for this issue?

**Comment by tolot27** [comment]
@berislavv I still not fully understand the issue. Please can you post screenshots of the problem, both from xDrip and Nightscout?

Does xDrip show the correct values for all historical data or only for the last 15 minutes?

**Comment by berislavv** [comment]
Its almost a year ago abaout the issue, i need to check on the phone if i can get so much history back, i changed the phone but i think the data are transfered.


Xdrip was setup to use Datricsae, also to send to nightscout display value. and to adjust historical values. When blucon is working everything was fine.
During the cirumstances we started to scan only NFC, no blucon. The kid is swiming, playing etc. and i dont scan via NFC for lets say one hour. After NFC scan on the xdrip graph everything looks fine, but on the Nightscout graph the last 15 minutes of data is  as the same on the xdrp but the rest 45 minutes is moved.
I was looking at the sql table in the phone and it looks for me that after the NFC scan the 15 minutes are sended to NIghtscout with Datricsae  calibration and the rest without.

Xdrip always show correct values, the issue i think is in the data uploader and where he gets the values to send to Nightscout.

We switched back to xdrip standard calibration few months ago, as he is now 7 years and growing, so now the xdrip standard calibration is good now.

Well i think this will be hard to trace, if you want i will try to get the data from last summer when i saw this, else i will close this issue.
